.github/workflows/verify-generation-config.yaml

```diff
@@ -9,35 +9,25 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@v4
-    - uses: actions/setup-java@v4
-      with:
-        java-version: 11
-        distribution: temurin
-        cache: maven
-    - uses: actions/setup-python@v5
-      with:
-        python-version: '3.11'
-    - name: copy generation scripts
+    - name: setup docker environment
       shell: bash
       run: |
-        # checkout `library_generation` directory from sdk-platform-java
-        git clone https://github.com/googleapis/sdk-platform-java.git
-        pushd sdk-platform-java
-        git pull
-        cp -r library_generation ../
-        popd
-        rm -rf sdk-platform-java
-    - name: install python dependencies
-      shell: bash
-      run: |
-        set -ex
-        pushd library_generation
-        pip install -r requirements.in
-        pip install .
-        popd
+        # we create a volume pointing to `pwd` (google-cloud-java) that will
+        # be referenced by the container and its children
+        if [[ $(docker volume inspect repo) != '[]' ]]; then
+          docker volume rm repo
+        fi
+        docker volume create --name "repo" --opt "type=none" --opt "device=$(pwd)" --opt "o=bind"
     - name: generate from configuration
       shell: bash
       run: |
         git config --global user.email "github-workflow@github.com"
         git config --global user.name "Github Workflow"
```

Comment:
Do we still need to set git user and email since they are already set in Dockerfile?

Reply:
We don't need to. I'll remove them